src/assets/scss/helpers.scss

```diff
@@ -89,6 +88,10 @@
   -moz-user-select: none;
   -ms-user-select: none;
   white-space: nowrap;
+  // ficxing button spacing issue. Fix- center aligning the buttons
+  display: inline-flex !important;
+  align-items: center;
+  justify-content: center;
```

Comment:
Add `line-height: normal`. Add a comment explaining what `normal` does and why it works here. Or try other values and explain why that might be better.

Adding `line-height` here will allow for the `<span>` tags to inherit the value and use it for the text part of the button.